test/gtest/uct/uct_test.cc

```diff
@@ -1441,6 +1453,8 @@ uct_test::mapped_buffer::~mapped_buffer() {
     m_entity.rkey_release(&m_rkey);
     if (m_mem.mem_type == UCS_MEMORY_TYPE_HOST) {
         m_entity.mem_free_host(&m_mem);
+    } else if (m_mem.mem_type == UCS_MEMORY_TYPE_RDMA) {
+        uct_mem_free(&m_mem);
```

Comment:
can we move this to m_entity.mem_free_host, for consistency? and rename m_entity.mem_free_host to m_entity.mem_free